integreat_cms/matomo_api/apps.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+from django.apps import apps, AppConfig
+from django.conf import settings
+
+logger = logging.getLogger(__name__)
+
+
+class MatomoApiConfig(AppConfig):
+    """
+    MatomoApiClient config inheriting the django AppConfig
+    """
+
+    name = "integreat_cms.matomo_api"
+
+    def ready(self):
+        """
+        Checking if API is available
+        """
+        # Only check availability if running a server
+        if apps.get_app_config("cms").test_external_apis:
+            if not settings.MATOMO_API_ENABLED:
+                logger.info("Matomo API is disabled")
+            elif settings.DEBUG:
+                logger.info(
+                    "Matomo API is enabled, but in debug mode. "
+                    "Push notifications are really sent, but only to the test region."
```

Comment:
Matomo is about statistics, not about push notifications. Also, there is no difference between debug mode and production mode for the Matomo API.
If you copy and paste code from other apps, please read the code at least once and think about whether renaming the variables is enough or whether there are other parts you need to adapt :smiley_cat: